packages/terra-clinical-detail-view/tests/jest/DetailList.test.jsx

```diff
@@ -55,12 +57,12 @@ it('should set the title text', () => {
 
 it('should set the heading level for the title', () => {
   const wrapper = shallow(defaultVariety);
-  expect(wrapper.html('.title')).toContain('<h2 class="title">Title</h2>');
+  expect(wrapper.html('.title')).toContain('<h2 class="title" id="list-title-id-00000000-0000-0000-0000-000000000000">Title</h2>');
```

Comment:
Can we also validate that aria-labelledby is being set correctly?

Reply:
Added here: https://github.com/cerner/terra-clinical/pull/890/commits/3417b2742d1cb5fbb9c56bc532abb50b68eba4d9